app/Http/Controllers/Frontend/Admin/EventController.php

```diff
@@ -57,8 +57,8 @@ public function renderEdit(int $id): View {
 
     public function denySuggestion(Request $request): RedirectResponse {
         $validated = $request->validate([
-                                            'id'      => ['required', 'exists:event_suggestions,id'],
-                                            'decline' => ['required', new Enum(EventRejectionReason::class)]
+                                            'id'            => ['required', 'exists:event_suggestions,id'],
+                                            'declineReason' => ['required', new Enum(EventRejectionReason::class)]
```

Comment:
Sorry, I'm just seeing it now. Do we call it `decline` or `reject`? Because the Enum is called `EventRejectionReason`.

Reply:
oh boy. I wanna go the easy route and rename the Enum, but reject is the better fitting word... 🥲